### ISSUE TITLE
msw init cli throws error "Unexpected token '?'"

### ISSUE THREAD

**Issue by codeAscetic** [kettanaito/msw#916]
## Describe the bug
When I execute command `npx msw init public/ --save` I get the following error: `Unexpected token '?'`.

## Environment

- `msw: 0.35.0`
- `nodejs: 12.18.2`
- `npm: 6.14.5`

**Comment by kettanaito** [comment]
Hey, @codeAscetic.

Thanks for reporting this. I can see that also failing one of our pull request builds (#913). We need to look into that. At first glance, it appears that something has changed in how the `inquirer` dependency is being read (contains illegal character).

**Comment by kettanaito** [comment]
The line of code that causes the issue is 

```js
/root/release/tmp/auto-update-worker/node_modules/inquirer/lib/prompts/rawlist.js:126
      this.selected = this.selected ?? -1;
```

However, `inquirer` package doesn't _have_ that line of code. Instead, this value assignment is written like this (same path):

```js
  getCurrentValue(index) {
    if (index == null) {
      index = this.rawDefault;
    } else if (index === '') {
      // No nullish-coalescing operators.
      index = this.selected;
    } else {
      index -= 1;
    }

    const choice = this.opt.choices.getChoice(index);
    return choice ? choice.value : null;
  }
```

It appears as if the already built `inquirer` gets transpired into a different build target somehow.

**Comment by kettanaito** [comment]
Digging further, I can see that the current dev environment installs `inquirer@8.1.1` while the latest public release installs `inquirer@8.1.5`. If we look at the same `getCurrentValue` function in 8.1.5 we can see something that begins to resemble the problematic code:

```js
  getCurrentValue(index) {
    if (index == null) {
      index = this.rawDefault;
    } else if (index === '') {
      // This can be coerced to "??" by some tooling.
      this.selected = this.selected === undefined ? -1 : this.selected;
      index = this.selected;
    } else {
      index -= 1;
    }
```

Strange enough, I can't reproduce the issue using the latest MSW (0.35.0) and NPM (6.14.8) or Yarn (1.22.10).

**Comment by kettanaito** [comment]
@codeAscetic, may I kindly ask you to submit a reproduction repository so I could investigate this?

**Comment by kettanaito** [comment]
Okay, I think I've found the root cause. 

`inquirer@8.1.3` introduced a bug in the `rawlist` module that resulted in the "Unexpected token ?" exception for any consumer (which MSW CLI is). You can see [inquirer@8.1.4](https://github.com/SBoudrias/Inquirer.js/releases/tag/inquirer%408.1.4) release notes that mention the fix for this issue. 

SInce `inquirer` is installed per semver, users that installed MSW in the window when 8.1.4 was the latest `inquirer` version got this issue. This issue won't be reproducible in any new installations, as the issue is fixed in `inquirer`.

**Comment by codeAscetic** [comment]
hi @kettanaito,

Thank for your quick response. 

Are you suggesting that I reinstall MSW, and that should in-turn install the latest dependency which contains the fix for this issue?

**Comment by kettanaito** [comment]
I suggest you re-install MSW. This will propagate the latest `inquirer`, which is 8.1.5. You can verify that via `npm ls inquirer` after the installation (you'll see it under the `msw` package).

**Comment by codeAscetic** [comment]
Thank you so much!!!

**Comment by codeAscetic** [comment]
hi @kettanaito, 

I have tried re-installing msw a few times after the merge so far, however it seems everytime i check with `npm ls inquirer` it ends up being the same as before `8.1.4` instead of `8.1.5` per your latest commit to msw version `0.35.0`. 

I am not too familiar with how commit updates work with package versioning, but so far I have tried updating my package.json by adding a `^` and a `~` in front of the msw version `0.35.0`, however it seems the re-installation doesn't pull in `inquirer` version `8.1.5`.

Would appreciate your advice on this matter.

Thank you for your time!
